NetCoreRestApi/WebAPI/Controllers/CategoriesController.cs

```diff
@@ -23,9 +23,10 @@ public CategoriesController(ICategoryManager categoryManager, IConverter<Categor
         [HttpGet]
         public IActionResult Get()
         {
-            var products = _categoryManager.GetAll();           
-            var productsDTO = _converter.ConvertFrom(products.ToList());
-            return Ok(productsDTO);
+            var categories = _categoryManager.GetAll();           
```

Comment:
it is better to use name like categroyModels, categoryEntites, categoryDtos so you will be able to understand type from name

Reply:
done